app/services/idv/session.rb

```diff
@@ -136,6 +136,24 @@ def user_phone_confirmation_session=(new_user_phone_confirmation_session)
       session[:user_phone_confirmation_session] = new_user_phone_confirmation_session.to_h
     end
 
+    def ssn_updated!
+      # Guard against unvalidated attributes from in-person flow in review controller
+      session[:applicant] = nil
+
+      invalidate_verify_info_step!
+      invalidate_phone_step!
+    end
```

Comment:
to me, the name was confusing here compared to what the method does, WDYT about `invalidate_steps_before_ssn!`?

Reply:
I can change it to `invalidate_steps_after_ssn!`